directives/btn-group/btn-group.component.js

```diff
@@ -0,0 +1,28 @@
+(function () {
+    'use strict';
+
+    angular.module('wfm.btnGroup', [])
+    .component('wfmBtnGroup',
+    {
+        templateUrl: 'directives/btn-group/btn-group.tpl.html',
+        controller: BtnGroupCtrl,
+        bindings: {
+            items: '<',
+            selected: '<',
+            output: '<',
+            btnClass: '<',
+            selectionClass: '<'
+        }
+    });
+
+    BtnGroupCtrl.$inject = [];
+    function BtnGroupCtrl() {
+        var ctrl = this;
+
+        ctrl.$onInit = function () {
+            if (ctrl.selected) {
```

Comment:
Not necessary to call `selected` in `onInit`? Is this an extra call? The parent component must already know the value of `selected` at this point.

Reply:
It is to select the "select" item passed in on load. if this is not done the item wont be marked until the output function is called on click